### DISCUSSION TITLE
Transfer .dcm to .nii.gz

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#5271] [Q&A]
Thanks for the great job!
I have some 3d medical images which are stored in dicom format, how to save that in .nii.gz by using MONAI?

**Comment by wyli** [comment]
for each input file name or directory, you can run:
```py
import monai.transforms as mt

img = mt.LoadImage(image_only=True, ensure_channel_first=True)("input_filename.dcm")
img = mt.SaveImage(output_ext="nii.gz", output_postfix="", resample=False)(img)
```

**Comment by a-parida12** [comment]
@YingdiZhang just a suggestion if you dont want to use monai. You could try this purposed built dcm--> nifti (https://pypi.org/project/nekton/) . it also allows you to got from nifti --> dicomseg too.
